doc/modules/feature_selection.rst

```diff
@@ -262,6 +260,55 @@ meta-transformer)::
     * :ref:`sphx_glr_auto_examples_ensemble_plot_forest_importances_faces.py`: example
       on face recognition data.
 
+.. _sequential_feature_selection:
+
+Sequential Feature Selection
+============================
+
+Sequential Feature Selection [sfs]_ (SFS) is available in the
+:class:`~sklearn.feature_selection.SequentialFeatureSelector` transformer.
+SFS can be either forward and backward:
```

Comment:
I am not a native speaker but `or` wouldn't be more appropriate?

Reply:
you're right, thanks

Reply:
and thanks for the review!